src/Message/AbstractResponse.php

```diff
@@ -26,13 +21,16 @@ public function __construct(RequestInterface $request, $data)
     public function getCode(): int
     {
         if (isset($this->data['code'])) {
-            return $this->data['code'];
+            return (int) $this->data['code'];
         }
 
-        return self::HTTP_OK;
+        return Response::HTTP_OK;
     }
 
-    public function getMessage(): string
+    /**
+     * @return array|string|null
```

Comment:
z tego co widziałem, to jak jest error to jest wartość i to jest string, jak nie ma wartości to poniżej zwracany jest pusty string więc null się nie pojawi, natomiast nie wiem skąd ten array ?
jak się pojawi array to może by go zwinąć do jednego stringa, żeby nie trzeba było żąnglować typem array|string ?